test/configs/test.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
```

Comment:
Would it be silly to just require the test.json file here so we don't have to keep the 2 files in sync? I mean, we probably won't be changing these much as the tests would break if we did. I dunno just thinking out loud.